highlight_view.py

```diff
@@ -656,12 +656,16 @@ def run(self):
         color: var(--yellowish);
         font-weight: bold;
     }
+    .copy {
+        margin-top: 8px;
+    }
```

Comment:
Sorry for not putting all the feedback in one go, but I’m also kinda just thinking aloud here and reacting from my phone...

So, iirc we can use em’s here instead of pixels and that would cause everything to scale nicely with font size. However, that’s not necessarily the best option. Perhaps we can steal what the scope name popup does that’s built into ST (i.e. it’s in the Default package so we can look at it, it’s not really documented or anything).

Anyway, if what I just wrote makes any sense to yourself out 😄 otherwise I’ll have a look at it myself later.

Reply:
Also a good call, I replaced `8px` with `0.5em` and it looks good regardless of how big or small you make ST's font.

Hopefully this is ready to merge now!

Reply:
Woohoo 🎉 thanks man! Great work 👍🏻